src/elements/select/select.ts

```diff
@@ -306,6 +321,46 @@ export class SbbSelectElement extends SbbUpdateSchedulerMixin(
     this._openPanelEventsController?.abort();
   }
 
+  protected override updateFormValue(): void {
+    let formValue: string | FormData | null;
```

Comment:
I don't think the common variable is necessary?
If you just inline the `this.internals.setFormValue` call into the if/else, it would also avoid the casting necessity.
What do you think?